test-tools/src/test/java/org/terracotta/org/junit/rules/TempFolderRuleTest.java

```diff
@@ -181,6 +187,34 @@ public void recursiveDeleteFolderWithZeroElements() throws IOException {
         assertFalse(folder.getRoot().exists());
     }
 
+    @Test
+    public void tempFolderIsOnlyAccessibleByOwner() throws IOException {
+        TemporaryFolder folder = new TemporaryFolder();
+        folder.create();
+
+        Set<String> expectedPermissions = new TreeSet<String>(Arrays.asList("OWNER_READ", "OWNER_WRITE", "OWNER_EXECUTE"));
+        Set<String> actualPermissions = getPosixFilePermissions(folder.getRoot());
+        assertEquals(expectedPermissions, actualPermissions);
+    }
+
+    private Set<String> getPosixFilePermissions(File root) {
+        try {
```

Comment:
Slight concern that the `Exception` catch is overly broad.  Could we just have a gating:
```
assumeThat(version, matches(regex));
```

Reply:
This is a direct port from JUnit 4.13.1 -- other than relocation, I'd prefer not to make any changes.  If there's a real issue, it'll need to be brought up with the JUnit crew.